MantidQt/CustomInterfaces/src/Reflectometry/QtReflEventView.cpp

```diff
@@ -80,13 +80,8 @@ std::string QtReflEventView::getTimeSlicingValues() const {
 * @return :: Time slicing type
 */
 std::string QtReflEventView::getTimeSlicingType() const {
-  std::map<SliceType, std::string> sliceTypeStrMap = {
-      {Uniform, "Uniform"},
-      {UniformEven, "UniformEven"},
-      {Custom, "Custom"},
-      {LogValue, "LogValue"}};
 
-  return sliceTypeStrMap[m_sliceType];
+  return m_sliceTypeStrMap.at(m_sliceType);
```

Comment:
As m_sliceTypeStrMap is now defined in the header file, it is essentially `const` after initialisation. We cannot use `[]` as that can also be used to insert new elements into the map.

I feel it is also worth mentioning that the code in question is only called once in reduction and is not performance-critical at all.